baselines/naming/src/v1beta1/naming_client.ts.baseline

```diff
@@ -807,124 +807,9 @@ export class NamingClient {
       options = optionsOrCallback as gax.CallOptions;
     }
     options = options || {};
-    this.initialize();
+    this.initialize1();
     return this.innerApiCalls.getProjectId(request, options, callback);
   }
-  function(
```

Comment:
FYI: The lines removed from the test baseline correspond to a feature (reserved words) that I'm not going to cover soon.